### PR TITLE
Changed linter config

### PR DESCRIPTION
**Pull request by RehReis**
## Description:
Changed linter configuration

## Context:
Changed linter configuration to not give error messages when importing JSX files.

## Checklist:
[x] changed .eslintrc.jsony file
[x] added package-lock.json to .gitignore

## Notes:
You may need to run 'git rm --cached package-lock.json' in order for git to stop identifying the file as changed.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -19,5 +19,6 @@
         "react"
     ],
     "rules": {
-    }
+    },
+    "import/extensions": {"jsx": "always"}
```
Comment (aaronfinsrud) [inline .eslintrc.json]: Cool! Good to know that this works.

File: .gitignore [hunk 2]
```diff
@@ -6,3 +6,5 @@ node_modules/
 
 # Config
 config.js
+
+package-lock.json
```
Comment (aaronfinsrud) [inline .gitignore]: I'm not super clear on the specifics, but this stackoverflow article recommends against adding the packag-lock.json to gitignore. 
https://stackoverflow.com/questions/48524417/should-the-package-lock-json-file-be-added-to-gitignore

**Review by aaronfinsrud — APPROVED** [review]
Looks good. You could consider keeping the package-lock.json out of the gitignore. That's all I've got for feedback. Cheers!

File: .gitignore (hunk 2 above)
Comment (RehReis) [reply .gitignore]: It actually didn't work. I didn't realize that this configuration disabled the linter until I started implementing a new component and saw that  I wasn't getting any linter error.
I changed the config again, and now it's working. 
Can you review it for me again, please?

File: .eslintrc.json [hunk 3]
```diff
@@ -19,5 +19,16 @@
         "react"
     ],
     "rules": {
+
+    },
+    "settings": {
```
Comment (aaronfinsrud) [inline .eslintrc.json]: This is to get rid of the red-squiggly lines under the .jsx imports, correct? I'm trusting you've tested it.

**Review by aaronfinsrud — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (RehReis) [reply .gitignore]: Hey @aaronfinsrud, just know I realized I answered the question about the linter where you were asking about the package-lock.json. 
I didn't know it was recommended to commit the package-lock.json as it changes every time you run 'npm install' and causes merging conflicts. I'll take it out of .gitignore as recommended.

File: .eslintrc.json (hunk 3 above)
Comment (RehReis) [reply .eslintrc.json]: Yes, that's right. The configuration I added before stopped the linter, but this new one keeps the linter working and gets rid of the 'error' message for jsx imports.

**Comment by RehReis** [comment]
Hey @aaronfinsrud,

I excluded package-lock.json from .gitignore as recommended. 
I also made the new changes I told you before, adding rules to the eslint (according to docs - https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/extensions.md) instead of using the import/resolver.

**Pull request merged**